atlast_sc_tests/unit_tests/test_calculator.py

```diff
@@ -88,42 +88,43 @@ def test_initialize_calculator_invalid(self, user_input, expected_raises):
             Calculator(user_input)
 
     @pytest.mark.parametrize(
-        'param,new_value,derived_params_recalculated,expected_raises',
+        'param,new_value,derived_params_recalculated,expected_raises,finetuned',
         [
             # User input
-            ('t_int', 1 * u.s, False, does_not_raise()),
-            ('sensitivity', 3 * u.mJy, False, does_not_raise()),
-            ('bandwidth', 10 * u.MHz, False, does_not_raise()),
-            ('n_pol', 1, False, does_not_raise()),
-            ('weather', 35, True, does_not_raise()),
-            ('elevation', 80 * u.deg, True, does_not_raise()),
-            ('obs_freq', 700 * u.GHz, True, does_not_raise()),
+            ('t_int', 1 * u.s, False, does_not_raise(),False),
```

Comment:
Exactly. I did not have time to compile a full set of tests for the broadband case, so I switched `finetune` to `False` during the tests to avoid issues with the pull request (and, thus, the default for `finetune` as `False`)